SU-HSDO/suhumsci
# Feat(STN-655): global message banner

# READY FOR REVIEW

## Summary
The Global Message allows “site managers to add a global message / alert to their websites. This site-wide message is displayed directly below the Stanford brand bar, and can be modified depending on the desired content and message type.

**The colors associated with the Global Message banner are independent of the color pairings**

- This work adds the styling for Global Message banner that is pulled from the campus product.
- This work also adds a new region to the Humsci Basic themes, including subthemes and places the blocks in those regions for both themes.

JIRA Card: https://sparkbox.atlassian.net/browse/STN-655

Campus Global Message Documentation: https://userguide.sites.stanford.edu/tour/site-organization-menus/managing-global-message

Campus Global Message Example: https://hsweb.sites.stanford.edu/

## Need Review By (Date)
asap

## Urgency
'medium'

## Steps to Test
1. npm run test
2. Import the block placement in the new Global Message regions: `lando drush @sparkbox_sandbox.local cim --partial`
_NOTE: This command will import the new configuration and override any current configuration you may have created, if you need to save that config you will want to export and commit that configuration before running that command._
3. Create a Global Message: http://sparkbox-sandbox.suhumsci.loc/admin/config/system/global-message (Configuration > Global Message)
4. Check Enable and add text and a link in the body as well as the other fields to verify the styling and placement.
5. Verify the different color messages on both themes.
6. Verify the messages all work properly using a Dark variant masthead by selecting the "Dark" option in the Appearance > Settings option.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
